R/Rules-methods.R

```diff
@@ -744,35 +730,50 @@ setMethod("maxDose",
               return(ret)
           })
 
+# nolint end
 
-## --------------------------------------------------
-## The maximum allowable number of dose levels method
-## --------------------------------------------------
-
-##' @describeIn maxDose Determine the maximum possible next dose based on
-##' maximum dose levels to increment for the next dose
-##'
-##' @example examples/Rules-method-maxDose-IncrementsNumDoseLevels.R
-setMethod("maxDose",
-          signature=
-            signature(increments="IncrementsNumDoseLevels",
-                      data="Data"),
-          def=
-            function(increments, data, ...){
-              ## determine what was the level of the last dose
-              lastDoseLevel <- tail(data@xLevel, 1)
-
-              ## determine the maximum next dose level
-              maxNextDoseLevel <- min(length(data@doseGrid),
-                                      lastDoseLevel + increments@maxLevels)
-
-              ## so the maximum next dose is
-              ret <- data@doseGrid[maxNextDoseLevel]
-
-              return(ret)
-            })
+# maxDose-IncrementsNumDoseLevels ----
 
+#' @rdname maxDose
+#' 
```

Comment:
just need to delete the one blank to satisfy the linter check
```suggestion
#'
```
similar below